### DISCUSSION TITLE
Displacy to SVG for span and entity visualization

### DISCUSSION THREAD

**Discussion by shmouelsamares** [explosion/spaCy#12298] [Help: Coding & Implementations] [answered]
Hi, I would like to export to an SVG file my Span style renders. 
I understand that only the dependency render is an SVG type of image.

Is there a way to export the Span renders? Or to customize the dependency visualizer to show spans? I need overlapping to be suppported.

**Comment by svlandeg** [comment, marked as answer]
Hi! You're right that only the dependency visualizer supports SVG directly because it's the best way to display the arcs. The NER and spans could be depicted with plain HTML though, so we chose that. Perhaps you can find a tool that converts HTML to SVG? (https://stackoverflow.com/questions/12005860/can-i-export-part-of-an-html-page-to-an-svg-image)
